workflows/iuf/operations/add-product-to-product-catalog.yaml

```diff
@@ -0,0 +1,49 @@
+#
+# MIT License
+#
+# (C) Copyright 2022 Hewlett Packard Enterprise Development LP
+#
+# Permission is hereby granted, free of charge, to any person obtaining a
+# copy of this software and associated documentation files (the "Software"),
+# to deal in the Software without restriction, including without limitation
+# the rights to use, copy, modify, merge, publish, distribute, sublicense,
+# and/or sell copies of the Software, and to permit persons to whom the
+# Software is furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included
+# in all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL
+# THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR
+# OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE,
+# ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR
+# OTHER DEALINGS IN THE SOFTWARE.
+#
+
+apiVersion: argoproj.io/v1alpha1
+kind: WorkflowTemplate
+metadata:
+  name: add-product-to-product-catalog
+spec:
+  entrypoint: main
+  templates:
+### Main Steps ###
+  - name: main
+    inputs:
+      parameters:
+      - name: global_params
```

Comment:
I recall Di saying that you have to supply auth_token also even if you are not using it. I am not sure about the validity of that statement. Anyway, mentioning it here in case you see errors.

Reply:
Thanks for the heads-up.

Reply:
With the current version on frigg, it was not required, I was able to successfully test this workflow template as written. Checking to see if there is still some reason that I might want to add it.

Reply:
We talked this over and slack and decided to leave it as-is. Unlike `local-path` this one doesn't seem to have any potential problem with validation of stages. If something changes we can definitely add it back in.